packages/notification/src/InlineNotification.tsx

```diff
@@ -3,6 +3,7 @@ import { HeadingProps } from "@kaizen/typography"
 import GenericNotification, {
   NotificationType,
 } from "./components/GenericNotification"
+import styles from "./components/GenericNotification.module.scss"
```

Comment:
You'll actually want to create the class in a `InlineNotification.module.scss` file since it's self-contained :)